Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -127,6 +127,7 @@ final class SchemesGenerator: SchemesGenerating {
             testTargets = [TestableTarget(target: targetReference)]
         } else {
             testTargets = graph.testTargetsDependingOn(path: project.path, name: target.name)
+                .sorted { $0.target.name < $1.target.name }
```

Comment:
Maybe it would be worth moving this to the `testTargetsDependingOn` method, or even in the place where we get targets? It might be useful in other places to have the target list stable.

Reply:
Thanks @vytis for the feedback.  Right, `Graph` looks like the right place for the sort operation 👍

I've put it `testTargetsDependingOn`, can move it to `targets(at path: AbsolutePath)` if we think it's actually better.

Something I'd consider:
- Maybe we can make `public func targets(at path: AbsolutePath) -> [TargetNode] ` private? looks it's used only in `Graph`
- Change types of all the computed properties derived from `Dictionary.values` (in `Graph` and maybe other places) to `Set` to express the fact the items are in random order (likely it has been discussed already, I might miss some context)